parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -33,7 +33,10 @@ ExecStart=/usr/local/bin/kubelet \
         --node-labels="${KUBELET_NODE_LABELS}" \
         --v=2 {{if NeedsContainerd}}--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock{{end}} \
         --volume-plugin-dir=/etc/kubernetes/volumeplugins \
-        $KUBELET_CONFIG \
+        {{- if IsDynamicKubeletSupported}}
+        --config /etc/default/kubeletconfig.json --dynamic-config-dir /etc/default/dynamickubelet \
```

Comment:
Just curious - do we need to pre-create the config dir or not?

Reply:
we don't need to, the directory will be created if not exist